cmd/node-joiner/testdata/add-nodes-overrides.txt

```diff
@@ -50,6 +50,7 @@ spec:
 status:
   agentLabelSelector: {}
   bootArtifacts:
+    discoveryIgnitionURL: ""
```

Comment:
I think there was a comment on that but cannot find it anymore, @eggfoobar where is used this field?